samples/Forms/LibVLCSharp.Forms.MediaElement/LibVLCSharp.Forms.Sample.MediaElement.iOS/AppDelegate.cs

```diff
@@ -26,5 +26,10 @@ public override bool FinishedLaunching(UIApplication app, NSDictionary options)
 
             return base.FinishedLaunching(app, options);
         }
+
+        public override UIInterfaceOrientationMask GetSupportedInterfaceOrientations(UIApplication application, [Transient] UIWindow forWindow)
+        {
+            return OrientationChangeListener.Subscribe(this);
```

Comment:
`GetSupportedInterfaceOrientations` is called every time the device orientation changes, causing multiple subscribes and potential leaks. That's not expected, right?

Reply:
`GetSupportedInterfaceOrientations`  works like that https://developer.apple.com/documentation/uikit/uiviewcontroller/1621435-supportedinterfaceorientations.
But in the `Subscribe` method, all subscriptions using MessagingCenter shoud be done one time.
I just fixed it.